composer.lock

```diff
@@ -1235,24 +1331,27 @@
                     "type": "tidelift"
                 }
             ],
-            "time": "2022-01-02T09:55:41+00:00"
+            "time": "2023-05-23T14:45:45+00:00"
         },
         {
             "name": "symfony/finder",
-            "version": "v6.0.11",
```

Comment:
This change is unexpected. Maybe dependencies not updated with PHP 8.0.x (min 8.0.2)?

If that was intended, this would remove PHP 8.0 and PHP 8.1 support at once.

Reply:
I had to pull in phpspec/prophecy in a higher version as a dev-dependency, otherwise the composer.lock file would not work with php 8.3:

 ```
 Problem 1
    - phpspec/prophecy is locked to version v1.16.0 and an update of this package was not requested.
    - phpspec/prophecy v1.16.0 requires php ^7.2 || 8.0.* || 8.1.* || 8.2.* -> your php version (8.3.4) does not satisfy that requirement.
  Problem 2
    - phpspec/prophecy v1.16.0 requires php ^7.2 || 8.0.* || 8.1.* || 8.2.* -> your php version (8.3.4) does not satisfy that requirement.
    - phpspec/phpspec 7.5.0 requires phpspec/prophecy ^1.9 -> satisfiable by phpspec/prophecy[v1.16.0].
    - phpspec/phpspec is locked to version 7.5.0 and an update of this package was not requested.
```

Reply:
`composer req --dev --with-dependencies` does the trick